falcon_pachinko/router.py

```
@@ -0,0 +1,96 @@
+
+        return route.path.format(**params)
+
+    async def on_websocket(
+        self, req: falcon.Request, ws: WebSocketLike
+    ) -> WebSocketResource:
+        """Handle a WebSocket connection dispatched to this router."""
+        mount_path = self._mount_path
+        if mount_path is None:
+            mount_path = req.uri_template
+            self._mount_path = mount_path
+        path = req.path[len(mount_path) :] or "/"
```

Flake8 (7.2.0)

[error] whitespace before ':'

(E203)